.azure-pipelines/unit-tests.yml

```diff
@@ -87,6 +70,7 @@ jobs:
         test/**/*.Tests.csproj
         test/benchmarks/**/*.csproj
         !src/Datadog.Trace.Tools.Runner/*.csproj
+        !test/Vendored.System.Diagnostics.DiagnosticSource.Tests/Vendored.System.Diagnostics.DiagnosticSource.Tests.csproj
```

Comment:
This is a left-over from when I was pushing this stuff to the `main` branch. Now I changed my mind and think that it will be better to remove this project from `main` and continue the work related to it on the branch that @macrogreg  worked on at the end of last year.

@macrogreg If you agree let's remove the vendored S.D.DS from `main`.

@pellared please keep this as it is on this PR and wait on an agreement if we are removing this from `main`.

Reply:
@pjanotti @macrogreg 
PR which removes the vedored stuff: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/52